src/main/java/org/project_kessel/client/HostsClient.java

```diff
@@ -4,32 +4,32 @@
 import io.grpc.stub.StreamObserver;
 import io.smallrye.mutiny.Uni;
 import io.smallrye.mutiny.operators.multi.processors.UnicastProcessor;
-import org.project_kessel.api.inventory.v1beta1.CreateRHELHostRequest;
-import org.project_kessel.api.inventory.v1beta1.CreateRHELHostResponse;
-import org.project_kessel.api.inventory.v1beta1.HostsServiceGrpc;
+import org.project_kessel.api.inventory.v1beta1.CreateRhelHostRequest;
+import org.project_kessel.api.inventory.v1beta1.CreateRhelHostResponse;
+import org.project_kessel.api.inventory.v1beta1.KesselRhelHostServiceGrpc;
 
 import java.util.logging.Logger;
 
 public class HostsClient {
```

Comment:
Can we keep these names consistent with protos?

Reply:
sure, I can update

Reply:
updated